src/views/ScorePage/ScorePage.js

```diff
@@ -48,7 +48,9 @@ function renderNickForm() {
   nickFormContainer.setAttribute('id', 'nickFormDiv');
   const input = document.createElement('input');
   input.setAttribute('id', 'nickname');
-  input.setAttribute('value', 'NICKNAME');
+  input.setAttribute('placeholder', 'NICK');
```

Comment:
Now placeholder can be confused with the name - Nick. Use "Enter your nickname..." instead of "NICK". Additionally, make placeholder less opaque and invasive